Mathlib/GroupTheory/SpecificGroups/ZGroup.lean

```diff
@@ -62,6 +66,10 @@ theorem of_surjective [Finite G] [hG : IsZGroup G] (hf : Function.Surjective f)
 instance [Finite G] [IsZGroup G] (H : Subgroup G) [H.Normal] : IsZGroup (G ⧸ H) :=
   of_surjective (QuotientGroup.mk'_surjective H)
 
+section Nilpotent
+
+variable (G)
```

Comment:
You only need G explicit in the new instance you're adding, right? If so, can you use `variable (G) in`?
As I understand it, explicit `variable`s are soft-discouraged: if there's an easy way to avoid this, we prefer that.

Reply:
Well, it's more that I felt like `G` ought to be explicit in `exponent_eq_card`. But I can do that manually if that's preferred.